src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -28,23 +26,74 @@ interface tableRowsChild {
   cells: { [title: string]: ReactNode }[]
 }
 
-interface TableStates {
-  page: number,
-  maxFeedId: number | undefined,
-  lastOffset: number,
-  lastPage: number,
-  storedPages: StudyInstanceWithSeries[][],
-  processingPluginIds: number[]
+type TableState = {
+  page: number, // Current table page number
+  maxFeedId: number | undefined, // ID of the latest Feed on Swift as of when PastAnalysisTable first mounted OR was last reset
+  lastOffset: number, // Page offset value for where to begin fetching the next unseen page
+  lastPage: number, // Table page number of the very last page (-1 means last page has not yet been seen)
+  storedPages: StudyInstanceWithSeries[][], // Stores pages that have been seen in an array of pages
+  processingPluginIds: number[] // Stores plugin ids associated with images that are currently processing, used for selective polling
+}
+
+const initialTableState: TableState = {
```

Comment:
I would suggest using all caps with underscores separating words for constants such as initial states.